evm/src/arithmetic/modular.rs

```diff
@@ -308,14 +350,16 @@ fn modular_constr_poly<P: PackedField>(
 
     // Verify that the output is reduced, i.e. output < modulus.
     let out_aux_red = &lv[MODULAR_OUT_AUX_RED];
-    // this sets is_less_than to 1 unless we get mod_is_zero when
+    // This sets is_less_than to 1 unless we get mod_is_zero when
     // doing a DIV; in that case, we need is_less_than=0, since the
     // function checks
     //
     //   output - modulus == out_aux_red + is_less_than*2^256
     //
     // and we were given output = out_aux_red
     let is_less_than = P::ONES - mod_is_zero * lv[IS_DIV];
+    // NB: output, modulus and out_aux_red are all in lv, so can just
+    // call yield_constr.constraint() in eval_ext_circuit_lt here.
```

Comment:
Oops, good catch!

Reply:
I've modified `eval_packed_generic_lt(...)` and everything it calls to use `yield_constr.constraint_transition(...)` rather than `yield_constr.constraint()`. This should fix the calls from `modular::`, but it does mean it will be calling `.constraint_transition()` in the `compare::` code even though those operations only use a single row. Are there any downsides to this that I might be missing?